.github/workflows/CI-tests.yml

```
@@ -49,7 +49,7 @@ jobs:
     needs: [build, lint]
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         django-version: ["3.2", "4.1", "4.2"]
```

According to https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django, the combination Django 3.2, Python 3.11 is not supported, 

Django version| Python versions
---|---
3.2|3.6, 3.7, 3.8, 3.9, 3.10 (added in 3.2.9)
4.0|3.8, 3.9, 3.10
4.1|3.8, 3.9, 3.10, 3.11 (added in 4.1.3)
4.2|3.8, 3.9, 3.10, 3.11
5.0|3.10, 3.11, 3.12
5.1|3.10, 3.11, 3.12

While this package is small, I don't really want to suggest things that may be misleading.

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
        django-version: ["3.2.9", "4.1.3", "4.2"]
        exclude:
            - django-version: "3.2"
              python-version: "3.11"
```

---

thank you for pointing this out, I have addressed it here [cc32b66](https://github.com/nimasmi/django-gov-notify/pull/8/commits/cc32b665d0dfc93de40ca7129168cbd0546b814c)